example/terraform_backend.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+export GOOGLE_PROJECT=terraform-test-hejda
+export TF_VAR_namespace=stage
+export TF_VAR_region=europe-west3
+export TF_VAR_zone=europe-west3-c
+export TF_VAR_vault_secret_path=secret/devops/internal/hejda-test
+export VAULT_ADDR=https://vault01.ack.ee
```

Comment:
Tohle není rozumné úplně veřejně vystavovat, zvlášť když jsme asi o rok pozadu s verzí Vaultu

Reply:
To sem nepiš :-D to nikdo nemusí vědět